controllers/openstackprovisionserver_controller.go

```diff
@@ -66,8 +68,16 @@ type OpenStackProvisionServerReconciler struct {
 	Scheme  *runtime.Scheme
 }
 
+// service account, role, rolebinding
+// +kubebuilder:rbac:groups="",resources=serviceaccounts,verbs=get;list;watch;create;update
+// +kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=roles,verbs=get;list;watch;create;update
+// +kubebuilder:rbac:groups="rbac.authorization.k8s.io",resources=rolebindings,verbs=get;list;watch;create;update
+// service account permissions that are needed to grant permission to the above
+// +kubebuilder:rbac:groups="security.openshift.io",resourceNames=privileged,resources=securitycontextconstraints,verbs=use
```

Comment:
Should we also remove L90-91 since we're granting the security context constraints at the cluster scope now?

Reply:
Sure, I did not notice those earlier.